IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/Cells/NewsArticleViewModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  NewsArticleViewModel.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+
+final class NewsArticleViewModel {
+    
+    let imageUrl: String
```

Comment:
We should use Bond's Observable<T> instead of just the type (i.e Observable<String>) so we can bind the properties on the View